tests/mt_api/mt_api_test.py

```diff
@@ -810,3 +810,135 @@ def test_deepl_bulk_mt_no_target_language(
                 or page_translation[target_language_slug].machine_translated is False
             )
+
+
+do_not_translate_title = [True, False]
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize("do_not_translate_title", do_not_translate_title)
+def test_do_not_translate_title(
+    load_test_data: None,
+    login_role_user: tuple[Client, str],
+    do_not_translate_title: bool,
+    settings: SettingsWrapper,
+    caplog: LogCaptureFixture,
+    mock_server: MockServer,
+) -> None:
+    """
+    Check `do_not_translate_title` flag works as expected
+
+    :param load_test_data: The fixture providing the test data (see :meth:`~tests.conftest.load_test_data`)
+    :param login_role_user: The fixture providing the http client and the current role (see :meth:`~tests.conftest.login_role_user`)
+    :param do_not_translate_title: The value of `do_not_translate_title` flag
+    :param settings: The fixture providing the django settings
+    :param caplog: The :fixture:`caplog` fixture
+    :param mock_server: The fixture providing the mock http server used for faking the DeepL API server
+    """
+    client, role = login_role_user
+
+    if role in [*PRIV_STAFF_ROLES, MANAGEMENT, EDITOR]:
+        region = Region.objects.get(slug=REGION_SLUG)
+        test_page = Page.objects.create(
+            region=region,
+            do_not_translate_title=do_not_translate_title,
+            lft=1,
+            rgt=2,
+            tree_id=14,
+            depth=1,
+        )
+        de_translation = PageTranslation.objects.create(
+            page=test_page,
+            title="xxxx",
+            slug="xxxx",
+            language=Language.objects.filter(slug="de").first(),
+            content="",
+        )
+
+        mt_setup(["de"], ["en-gb", "en-us"], ["en"], ["ar"], settings, mock_server)
+
+        with patch.object(
+            GoogleTranslateApiClient,
+            "__init__",
+            setup_fake_google_translate_api,
+        ):
+            edit_page_de = reverse(
+                "edit_page",
+                kwargs={
+                    "region_slug": REGION_SLUG,
+                    "language_slug": "de",
+                    "page_id": test_page.id,
+                },
+            )
+            client.post(
+                edit_page_de,
+                data={
+                    "title": "Neuer Titel",
+                    "content": "Neuer Inhalt",
+                    "mirrored_page_region": "",
+                    "automatic_translation": "on",
+                    "mt_translations_to_create": Language.objects.filter(slug="en")
+                    .first()
+                    .id,
+                    "status": status.PUBLIC,
+                    "_position": "right",
+                    "do_not_translate_title": do_not_translate_title,
+                },
+            )
+
+            de_translation = PageTranslation.objects.filter(
+                page=test_page, language__slug="de"
+            ).first()
```

Comment:
```suggestion
            de_translation = PageTranslation.objects.get(
                page=test_page, language__slug="de"
            )
```

Reply:
For the translations I had to go back to `.filter().first()` as there are multiple translations (because they are "updated" while doing MT) and `MultipleObjectsReturned` is thrown.